solutions/standard/variables.tf

```diff
@@ -35,6 +35,15 @@ variable "ssh_public_key" {
   }
 }
 
+variable "machine_type" {
+  type = string 
+  description = "input machine type: valid values are: bz2-4x16, bz2-8x32, bz2-16x64, cz2-8x16, cz2-16x32, mz2-2x16, mz2-4x32, mz2-8x64, mz2-16x128"
+  validation {
+    condition  = contains(["mz2o-2x16","bz2-4x16", "bz2-8x32", "bz2-16x64", "cz2-8x16", "cz2-16x32", "mz2-2x16", "mz2-4x32", "mz2-8x64", "mz2-16x128"], var.machine_type)
+    error_message = "Valid values for machine_type are: mz2o-2x16, bz2-4x16, bz2-8x32, bz2-16x64"
```

Comment:
Update your error message to contain all the same valid machine type values as in the description. Also - remove "mz2o-2x16" as that is an internal profile and most accounts will not have.

Reply:
Made changes as suggested.